Doc/library/dis.rst

```diff
@@ -1114,8 +1114,13 @@ All of the following opcodes use their arguments.
 .. opcode:: RAISE_VARARGS (argc)
 
    Raises an exception. *argc* indicates the number of arguments to the raise
-   statement, ranging from 0 to 3.  The handler will find the traceback as TOS2,
-   the parameter as TOS1, and the exception as TOS.
+   statement, ranging from 0 to 2. These parameter counts correspond to the 3
+   different forms of the ``raise`` statement:
+
+   * ``0``: ``raise`` (re-raise previous exception).
+   * ``1``: ``raise TOS`` (raise exception instance or type at ``TOS``).
```

Comment:
Do we need to add full stop here? It looks a bit weird to me :)

Reply:
Do you mean the final 'dot' at the end of the sentence?

Reply:
Yes, shouldn't it be inside the parenthesis?

```
* ``0``: ``raise`` (re-raise previous exception.)
```

or

```
* ``0``: ``raise`` (re-raise previous exception)
```

Personally, I prefer the latter form but I'm not a native English speaker. @csabella what do you think?

Reply:
@csabella what do you think about the full stop?